BUILD.bazel

```diff
@@ -30,27 +30,64 @@ cc_library(
         "stout/closure.h",
         "stout/compose.h",
         "stout/catch.h",
-        "stout/event-loop.h",
-        "stout/timer.h",
         "stout/head.h",
-        "stout/filesystem.h",
         "stout/parallel.h",
         "stout/iterate.h",
-        "stout/dns-resolver.h",
-        "stout/signal.h",
         "stout/collect.h",
         "stout/filter.h",
         "stout/take.h",
     ],
     srcs = [
         "stout/scheduler.cc",
         "stout/static-thread-pool.cc",
-        "stout/event-loop.cc",
     ],
     deps = [
         "@com_github_google_glog//:glog",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
```suggestion
    visibility = ["//visibility:private"],
```

Reply:
Sounds good, I'll make the sub-libraries private!